spinnaker/data_source_spinnaker_application.go

```diff
@@ -35,6 +36,6 @@ func datasourceApplication() *schema.Resource {
 				Computed: true,
 			},
 		},
-		Read: resourceSpinnakerProjectRead,
+		ReadContext: resourceSpinnakerApplicationRead,
```

Comment:
Just question, the previous implementation was incorrect, right? ( It should have been `Application`, not `Project` )

Reply:
Yes, it was incorrect 🙇